getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,56 @@
+Not all hosts in your environment run pods/virtualized workloads (i.e. containers managed by Kubernetes or OpenShift, or VMs managed by OpenStack). You may also have physical machines or legacy applications that need to securely communicate with pods, which cannot be move into an orchestrated cluster. You can secure these hosts (called **non-cluster hosts**) by installing {{site.prodname}} networking and network policy, or only network policy. The advantage of using {{site.prodname}} to protect the host are:
 
+- You get all of the {{site.prodname}} networking options: BPG, or overlay with VXLAN, IP-in-IP
+- You can use the same {{site.prodname}} policy configuration as you do with pods  
+    For example, you can write a cluster-wide policy for hosts that is immediately applied to every host. Use one tool to protect both hosts and pods. For details on how to restrict traffic to/from hosts using {{site.prodname}} network policy see, [Protect hosts]({{site.baseurl}}/security/protect-hosts)
 
 ### Concepts
 
-#### Workloads and Hosts
+A **non-cluster host** is a computer that "hosts" pods in a cluster, or a computer that directly run applications that is not part of a cluster. Using {{site.prodname}}, you can secure the host network interfaces called, **host endpoints** (to distinguish them from pods/**workload endpoints**). {{site.prodname}} provides these options for protecting non-cluster hosts.
 
-We use the term workload to mean a pod or VM running as a guest on a computer with {{site.prodname}} installed. If your cluster is a Kubernetes or OpenShift cluster, workloads are pods, if your cluster is an OpenStack cluster, workloads are VMs. 
 
-We use the term host to mean a computer where {{site.prodname}} is installed. These include computers that are part of a cluster and “host” workloads, as well as computers that are not part of the cluster and run applications directly, which we call "non-cluster" hosts.  {{site.prodname}} does not handle the networking from host to host ({{site.prodname}} assumes this is set up), but it can be used to handle networking between hosts and workloads.  {{site.prodname}} can also provide network policy for hosts, regardless of whether or not the hosts run any workloads.  This guide focuses on non-cluster hosts.
 
-#### {{site.prodname}} networking on non-cluster hosts
 
-When {{site.prodname}} networking is enabled, {{site.prodname}} provides the virtual networking for the workloads in your cluster, allowing them to communicate with one another. It is also responsible for setting up networking so that hosts can communicate with workloads and vice versa.  {{site.prodname}} does not handle networking for host to host communications.
+**Note**: 
+{{site.prodname}} does not handle host-to-host networking; {{site.prodname}} assumes this is already set up. 
```

Comment:
I like the original phrase " ... hosts can communicate with workloads and vice versa." But I agree the original language is convoluted. Perhaps replace the "host-to-host networking" - which isn't quite correct, with "host to workload" or similar?

Reply:
Changed to "host to workload"